### PR TITLE
Create 2016-08-27-Shift

### PR DESCRIPTION
**Pull request by kishoian**

### CODE REVIEW

File: content/attacks/posts/2016-08-27-Shift [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+date: 2016-08-27
+custodians: Shift
+categories: 51%
+title: Shift (SHIFT) experiences a 51% attack in August 2016
```
Comment (jhirschkorn) [inline content/attacks/posts/2016-08-27-Shift:5]: quotes on title and .md in file name

File: content/attacks/posts/2016-08-27-Shift [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+## Summary
+
+Shift, a blockchain project focused on decentralized web hosting, was hit by a 51% attack in late August 2016. The attackers were able to take control of more than half of the network's mining power, enabling them to reorganize the blockchain and carry out double-spending attacks.
+
+## Attackers
+
+The identities of the attackers behind the Shift 51% attack remain unknown. However, they were likely individuals or a group with substantial resources to amass a significant amount of mining power to execute the attack.
```
Comment (jhirschkorn) [inline content/attacks/posts/2016-08-27-Shift:14]: delete the second sentence. dont speculate

File: content/attacks/posts/2016-08-27-Shift [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+## Losses
+
+The exact amount of losses in the Shift 51% attack is not clear. However, such attacks usually result in financial losses as attackers can double-spend tokens, which affects the integrity and trust of the network.
```
Comment (jhirschkorn) [inline content/attacks/posts/2016-08-27-Shift:18]: delete the second sentence

**Review by jhirschkorn — CHANGES_REQUESTED** [review]

**Review by evgenydmitriev — APPROVED** [review]

**Review by evgenydmitriev — APPROVED** [review]

**Pull request merged**